src/hiero_sdk_python/consensus/topic_create_transaction.py

```
@@ -122,122 +38,52 @@ def set_auto_renew_period(self, seconds: Union[Duration, int]) -> "TopicCreateTr
+            autoRenewAccount=self.auto_renew_account.to_proto() if self.auto_renew_account is not None else None,
+            memo=self.memo
+        ))
+
         return transaction_body
-    def build_scheduled_body(self) -> SchedulableTransactionBody:
-        """
-        Builds the scheduled transaction body for this topic create transaction.
 
-        Returns:
-            SchedulableTransactionBody: The built scheduled transaction body.
+    def _execute_transaction(self, client, transaction_proto):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `_execute_transaction`

(ANN202)